src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -63,13 +62,16 @@ class ReportTypingIndicator extends React.Component {
 
             case 1:
                 return (
-                    <TextWithEllipsis
-                        leadingText={PersonalDetails.getDisplayName(this.state.usersTyping[0])}
-                        trailingText={` ${this.props.translate('reportTypingIndicator.isTyping')}`}
-                        textStyle={[styles.chatItemComposeSecondaryRowSubText]}
-                        wrapperStyle={[styles.chatItemComposeSecondaryRow]}
-                        leadingTextParentStyle={styles.chatItemComposeSecondaryRowOffset}
-                    />
+                    <Text
+                        style={[
+                            styles.chatItemComposeSecondaryRowSubText,
+                            styles.chatItemComposeSecondaryRowOffset,
+                        ]}
+                        numberOfLines={1}
+                    >
+                        {PersonalDetails.getDisplayName(this.state.usersTyping[0])}
+                        {` ${this.props.translate('reportTypingIndicator.isTyping')}`}
+                    </Text>
```

Comment:
Curious to know why `TextWithEllipsis` was removed and replaced with `Text`?

Reply:
Did you run the tests of PR which added `TextWithEllipsis` after removing it? It causes the following issue.
![image](https://user-images.githubusercontent.com/24370807/193561680-c3a4c603-9960-4cf5-903e-9cf7b3234fbf.png)

Reply:
If the text is large enough in small screens. There will be two .... I have seen this issue logged. Don't remember the issue id.

Reply:
That is a different story. But this change will break the web as well.

Reply:
> Did you run the tests of PR which added TextWithEllipsis after removing it? It causes the following issue.
> image
> is Typing is trimmed.

I am sorry. I did not understand. Could you elaborate it?

Reply:
The issue is that `is typing...` is getting trimmed when a long email is used. But I see that it is happening on the main as well. Possibly a regression. 

But we can't remove the `TextWithEllipsis` even though it not working as expected. `TextWithEllipsis` was created to handle this specific case only to trim the leading text and keep the trailing text fully visible all the time.  

I understand that currently, `TextWithEllipsis` not working correctly here (due to styling changes) but if we accept your change, it will never work. 

So eventually, your changes will be reverted as well.

we should find a way to fix this issue without removing the old `TextWithEllipsis` code.

Reply:
Thanks for clarifying, should I change it back to `TextWithEllipsis`? At least both will use the same component to display the typing status.